### ISSUE TITLE
Modal: Flacky test

### ISSUE THREAD

**Issue by cvanderleij-ilionx** [dso-toolkit/dso-toolkit#3454]
## Omschrijving van de taak

De volgende it faalt zo nu en dan in modal.cy.ts

1) should have keyboard accessible body container

**Comment by cvanderleij-ilionx** [comment]
Deze is gefixed in de release commit door het toevoegen van `.hydrated` class aan de `.dso-modal` selectors in de cy test
